odd-platform-ui/src/components/DataModelling/Relationships/RelationshipsSearchInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useCallback } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { SearchInput } from 'components/shared/elements';
+
+interface Props {
+  value?: string;
+}
+
+const RelationshipsSearchInput = ({ value }: Props) => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const handleSearch = useCallback(
```

Comment:
I'm not sure, but do we need to debounce this handler?

Reply:
Usually we do but not in this case due it uses `SearchInput` component and `handleSearch` triggers not by input change but by mouse/keyboard event.